### ISSUE TITLE
No Sound After Update to 0.6.5-1302

### ISSUE THREAD

**Issue by janagyjr** [MultiMC/MultiMC5#2545]
System Information
-----------------------------
MultiMC version:
0.6.5-1302

Operating System:
Windows 10 Pro 1803

Summary of the issue or suggestion:
----------------------------------------------
In the last version of MultiMC (0.6.4) had no issues with the sounds in Minecraft, upon launching an instance after upgrading to 0.6.5-1302 through the built-in upgrade mechanism Minecraft is unable to play any sounds. 

What should happen:
------------------------------
Minecraft should play sounds through my listening device.

Steps to reproduce the issue (Add more if needed):
-------------------------------------------------------------
1. Install 0.6.4, launch a modpack, enjoy sound.

2. Upgrade to 0.6.5-1302, launch Minecraft

3. ???

Suspected cause:
---------------------------
I'm not a programmer. I have no clue.

Logs/Screenshots:
----------------------------
[//]: # (Please refer to https://github.com/MultiMC/MultiMC5/wiki/Log-Upload for instructions on how to attach your logs.)

Screenshots are useless for a sound issue and there is no obvious way to actually push an upload of the logs.

Additional Info:
---------------------------
I'm using the stable branch for upgrades

**Comment by janagyjr** [comment]
Sorry, restarting MultiMC seems to have fixed it.

**Comment by janagyjr** [comment]
Here is a gist of the log, the issue has returned as described above. Restarting MultiMC only makes the issue intermittent. 
https://gist.github.com/janagyjr/55e7c108380e698a56764d574b94b3ed

**Comment by peterix** [comment]
```
[19:05:44] [Sound Library Loader/INFO] [minecraft/SoundManager]: Starting up SoundSystem...
[19:05:45] [Thread-17/INFO] [minecraft/SoundManager]: Initializing LWJGL OpenAL
[19:05:45] [Thread-17/INFO] [minecraft/SoundManager]: (The LWJGL binding of OpenAL.  For more information, see http://www.lwjgl.org)
AL lib: (EE) MMDevApiOpenPlayback: Device init failed: 0x80004005
AL lib: (EE) MMDevApiOpenPlayback: Device init failed: 0x80004005
[19:05:45] [Thread-17/ERROR] [minecraft/SoundManager]: Error in class 'LibraryLWJGLOpenAL'
[19:05:45] [Thread-17/ERROR] [minecraft/SoundManager]: Unable to initialize OpenAL.  Probable cause: OpenAL not supported.
[19:05:45] [Thread-17/WARN] [minecraft/SoundManager]: ERROR MESSAGE:
[19:05:45] [Thread-17/INFO] [minecraft/SoundManager]: Could not locate OpenAL library.
[19:05:45] [Thread-17/WARN] [minecraft/SoundManager]: STACK TRACE:
[19:05:45] [Thread-17/INFO] [minecraft/SoundManager]: org.lwjgl.openal.AL.create(AL.java:156)
[19:05:45] [Thread-17/INFO] [minecraft/SoundManager]: org.lwjgl.openal.AL.create(AL.java:102)
[19:05:45] [Thread-17/INFO] [minecraft/SoundManager]: org.lwjgl.openal.AL.create(AL.java:206)
[19:05:45] [Thread-17/INFO] [minecraft/SoundManager]: paulscode.sound.libraries.LibraryLWJGLOpenAL.init(LibraryLWJGLOpenAL.java:164)
[19:05:45] [Thread-17/INFO] [minecraft/SoundManager]: paulscode.sound.SoundSystem.CommandNewLibrary(SoundSystem.java:1576)
[19:05:45] [Thread-17/INFO] [minecraft/SoundManager]: paulscode.sound.SoundSystem.CommandQueue(SoundSystem.java:2572)
[19:05:45] [Thread-17/INFO] [minecraft/SoundManager]: paulscode.sound.CommandThread.run(CommandThread.java:121)
[19:05:45] [Client thread/INFO] [FML]: OBJLoader.Parser: command 's' (model: 'advancedrocketry:models/block/models/advrocketmotor.obj') is not currently supported, skipping. Line: 347 's off'
[19:05:45] [Sound Library Loader/WARN] [minecraft/SoundManager]: ERROR MESSAGE:
[19:05:45] [Sound Library Loader/INFO] [minecraft/SoundManager]: Could not locate OpenAL library.
[19:05:45] [Sound Library Loader/INFO] [minecraft/SoundManager]: Starting up SoundSystem...
[19:05:45] [Thread-19/INFO] [minecraft/SoundManager]: Switching to No Sound
[19:05:45] [Thread-19/INFO] [minecraft/SoundManager]: (Silent Mode)
[19:05:46] [Sound Library Loader/INFO] [minecraft/SoundManager]: Sound engine started
```

This looks like an issue with your system, not with the launcher or the game.

**Comment by peterix** [comment]
Or maybe...

http://openal.org/pipermail/openal/2017-May/000635.html
http://openal.org/pipermail/openal/2017-May/000636.html
http://openal.org/pipermail/openal/2017-May/000637.html
https://bugs.mojang.com/browse/MC-9974
https://github.com/LWJGL/lwjgl/issues/117

**Comment by peterix** [comment]
Therefore, it's a vanilla bug. And there's something slightly off about your system that triggers it.

**Comment by janagyjr** [comment]
Stock Windows 10 Pro install, stock hardware (aside from two additional hard drives). Anyway, thanks for the time.

**Comment by peterix** [comment]
Stock hardware is meaningless.
Windows 10 is also meaningless. It updates so often that it could break and fix itself within a few weeks.

Let's keep this open.

**Comment by unknowntrojan** [comment]
Same issue here. 

Windows 10 LTSC Build 17763
AMD A8-6600K APU overclocked from 3.8 GHz to 4.4 GHz (and surprisingly, stable)
NVIDIA GeForce GTX 1050 Ti (only slightly overclocked as the bottleneck is always my CPU, if any)
16 GB RAM (DDR3-1333)
MultiMC on one of my SSD's (plenty of space)


Tried deleting the libraries folder multiple times, and the openal lib in com.paulscode does exist, but minecraft says it can't find it. Downgrading LWJGL to 2.9.3 fixed it, but deleting 2.9.4-nightly and letting it redownload still throws the same error.

**Comment by janagyjr** [comment]
F3+T is a good workaround, though I haven't had this issue on launch in a while.
